src/components/PrimeIntro.vue

```diff
@@ -121,31 +122,48 @@ export default class PrimeIntro extends Vue {
     font-size: 48px;
     font-weight: 800;
     line-height: 58px;
-    .margin-bottom(3);
+    .margin-bottom(2.5);
 
     &--subtitle {
       .margin-top(5);
       font-weight: bold;
       font-size: 20px;
     }
+
+    &--bold {
+      font-family: "Barlow";
+      font-weight: 900;
+    }
   }
 
   &__list {
-    font-size: 12px;
+    list-style: none;
+    margin-block-start: 0;
+    padding-inline-start: 0;
+
     &__item {
       font-weight: 500;
       font-size: 20px;
       line-height: 23px;
       .margin-bottom(1.5);
+      color: @dark-6;
+      .flex();
+      align-items: center;
 
       i {
         .margin-right(2);
       }
     }
+
+    &__price {
+      font-size: 12px;
+      font-weight: 500;
+    }
   }
 
   &__button {
     .margin-top(5);
+    width: 176px;
```

Comment:
remove `width: 176px` 
add this code instead 
`.s-button {
      width: 176px;
      @media (max-width: 680px) {
        width:   100%;
      }
    }`